ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -776,6 +776,31 @@ def test_tensorflow_shape_n(
     )
 
 
+# size
+@handle_frontend_test(
+    fn_tree="tensorflow.size",
+    aliases=["tensorflow.raw_ops.Size"],
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"), max_num_dims=4
+    ),
+    output_dtype=st.sampled_from(["int32", "int64"]),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_size(  # NOQA
+    *, dtype_and_x, frontend, test_flags, fn_tree, on_device, output_dtype
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
+        out_type=output_dtype,
+    )
+
+
```

Comment:
Firstly remove `aliases=["tensorflow.raw_ops.Size"],`, raw_ops aren't exactly aliases since they accept only kwargs so we don't treat them exactly as such (and hence also have tests for them).
Please remove the `# NOQA` and arange all arguments in seperate lines like all other tests
The reason the test fails for this `size` is because tf does not accept anythng other than tf dtypes for `out_type` here. Not sure why there's this difference... I would recommend you just don't generate the `out_type` arg since it will inevitably cause failures and just leave a comment explaining that at the top of the test